js/src/Figure.ts

```diff
@@ -327,6 +329,10 @@ export class Figure extends widgets.DOMWidgetView {
       window.removeEventListener('resize', this.debouncedRelayout);
     });
 
+    if (this.model.get('show_toolbar')) {
```

Comment:
We should maybe handle the case when `show_toolbar` is dynamically updated. Something like:

```typescript
this.model.on('change:show_toolbar', () => {
  if (this.model.get('show_toolbar')) {
    this.create_toolbar();
  } else {
    this.remove_toolbar();
  }
});
```